api/src/Authentication/Type/OIDC.php

```diff
@@ -56,7 +59,12 @@ function authenticate($login, $password)
     function check()
     {
         global $cookie_key;
-        return($this->getUser($_COOKIE[$cookie_key]));
+
+        if (array_key_exists("cookie_key", $_COOKIE)) {
```

Comment:
```suggestion
        if (array_key_exists($cookie_key, $_COOKIE)) {
```

Reply:
As per PHP docs, the key is a string or an int:

https://www.php.net/manual/en/function.array-key-exists.php

` array_key_exists(string|int $key, array $array): bool`